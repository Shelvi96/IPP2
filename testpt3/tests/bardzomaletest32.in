NEW ID

04308818:>02
62;;38>;9;336:73
5460821>:;9
508738306>99914
0>::78:4
;:0276702>32:3
4>:2:62250
5>39:2;25
8462;:515>:27401
3>:7
::50146>22618631;
;>;6130;6392
;4>;94576;
@80
@13;412;6536467:5721:054
@375379591856:408;609222187;047
@0393:;0;095:71:3597
@3194:83:5;25640830079
@59:34704:75;;773
@06490
@90320:39300;57319;:8:
@45883067178498316:5
@9:553
@42368;0887
@6311;:3;54939
