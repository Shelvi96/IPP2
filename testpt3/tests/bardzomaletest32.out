0
2010101000
386585841919131648
36864
74922926
72
0
21100100
100110
0
0
0
